### PR TITLE
Refactor code and update UserGuide

### LINKED ISSUE #61: Code Review for @WooKaiNing
`src/main/java/cashleh/parser/Parser.java`
* Line 235, 236, 237: Consider assigning these values to variables then referencing when needed to reduce usage of magic strings
* Line 265: for this if conditional block, consider perhaps abstracting it out to a separate method, so as to improve the readability filterBy() through practicing single layer of abstraction

`src/main/java/cashleh/transaction/ExpenseStatement.java`
* Likewise, consider assigning the "||" delimiter to a variable then referencing as needed to reduce usage of magic literals.

`src/main/java/cashleh/transaction/IncomeStatement.java`
* Similar issue as the one above

### PR DESCRIPTION
**Pull request by WooKaiNing**
Fixes #61

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -37,6 +37,22 @@ Format: `addIncome DESCRIPTION /amt AMOUNT /date DATE /cat CATEGORY`
 Example of usage: 
 
 `addIncome monthly salary /amt 2500 /date 30/09/2023`
+```
+    ____________________________________________________________
+	The following income was added:
+	Income: monthly salary (Amount: 2500.0, Date: 30/09/2023)
+	____________________________________________________________
```
Comment (BenedictChannn) [inline docs/UserGuide.md]: I think there is some spacing issue for the second horizontal line?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -100,6 +149,17 @@ Format: `viewExpenses`
...
+|    ID    |     Type     |        Date        |         Description          |      Category      |       Amount       |
++----------+--------------+--------------------+------------------------------+--------------------+--------------------+
+|    1     |   Expense    |     2023-09-30     |           milk tea           |     FOOD_DRINK     |       - $2.5       |
++-----------------------------------------------------------------------------------------------------------------------+
+| Total Expense: $2.5                                                                                                   |
++-----------------------------------------------------------------------------------------------------------------------+
+```
 
 ### Viewing the entire financial statement: `viewFinancialStatement`
 Shows the net cash on hand and lists each income and expense record along with its description, amount, date and (optional) category.
```
Comment (BenedictChannn) [inline docs/UserGuide.md:165]: Can probably mention that the financial statement displayed will be sorted by date

File: src/main/java/cashleh/parser/Parser.java [hunk 3]
```diff
@@ -51,8 +51,9 @@ public class Parser {
...
     private static final String FILTER_INCOME = "filterIncome";
     private static final String FILTER = "filter";
-
-
+    private static final String AMOUNT_SUFFIX = "/amt";
+    private static final String DATE_SUFFIX = "/date";
+    private static final String CATEGORY_SUFFIX = "/cat";
 
     private static final String AMT_KEYWORD = "/amt";
     private static final String DATE_KEYWORD = "/date";
```
Comment (BenedictChannn) [inline src/main/java/cashleh/parser/Parser.java]: Can we make use of the AMT_KEYWORD and DATE_KEYWORD?

**Review by mcmc101001 — APPROVED** [review]
LGTM

**Review by BenedictChannn — APPROVED** [review]
LGTM

**Pull request merged**